### PR TITLE
[SMALLFIX] javadoc for FileInStream#getUnderStoreStreamFactory

### PR DESCRIPTION
**Pull request by ParanoiaW**

### CODE REVIEW

File: core/client/src/main/java/alluxio/client/file/FileInStream.java [hunk 1]
```diff
@@ -345,6 +345,14 @@ protected InputStream createUnderStoreBlockInStream(long blockStart, long length
         getUnderStoreStreamFactory(path, mContext));
   }
 
+  /**
+   *Creates and returns a {@link UnderStoreStreamFactory} for the UFS.
```
Comment (ifcharming) [inline core/client/src/main/java/alluxio/client/file/FileInStream.java]: NIT: `*Creates` -> `* Creates`

**Comment by ParanoiaW** [comment]
jenkins, please test

**Comment by yupeng9** [comment]
@ParanoiaW can you resolve the conflicts?

**Comment by ParanoiaW** [comment]
OK, the conflicts have been resolved

**Comment by saltylin** [comment]
@ParanoiaW can you resolve the conflicts?

**Comment by yupeng9** [comment]
@ParanoiaW can you resolve the conflicts?

**Pull request closed without merging**